src/apps/components/AppPage/AppPageNav.tsx

```diff
@@ -12,16 +13,23 @@ interface AppPageNavProps {
   supportUrl: string | undefined | null;
   homepageUrl: string | undefined | null;
   author: string | undefined | null;
+  appId: string;
 }
 
 export const AppPageNav: React.FC<AppPageNavProps> = ({
   name,
   supportUrl,
   homepageUrl,
   author,
+  appId,
 }) => {
   const location = useLocation<LinkState>();
   const goBackLink = location.state?.from ?? AppUrls.resolveAppListUrl();
+  const navigate = useNavigator();
+
+  const navigateToManageAppScreen = useCallback(() => {
```

Comment:
**question**: the same question as for `useMemo` - do we need to wrap it in `useCallback`?

Reply:
only small perf improvement, can be deleted